pkgs/top-level/all-packages.nix

```diff
@@ -1423,6 +1423,8 @@ in
 
   dislocker = callPackage ../tools/filesystems/dislocker { };
 
+  distgen = python3Packages.callPackage ../development/tools/distgen {};
```

Comment:
This can't be easily overridden. Can you use `python3.pkgs` in the expression?

Reply:
I'm not sure what you mean. I just replicated how other python3 based tools are packaged.

Reply:
```suggestion
  distgen = callPackage ../development/tools/distgen {};
```

```nix
{ python3 ...

python3.pkgs.buildPythonApplication rec { ...
```